packages/date-picker/src/panel/date-range.vue

```diff
@@ -426,9 +426,8 @@
       },
 
       handleChangeRange(val) {
-        const defaultTime = this.defaultTime || [];
-        this.minDate = modifyWithGivenTime(val.minDate, defaultTime[0]);
-        this.maxDate = modifyWithGivenTime(val.maxDate, defaultTime[1]);
+        this.minDate = val.minDate;
```

Comment:
I think it's better to "trim" time before passing {min, max}Date to `date-table`.

Reply:
I cancelled my previous changes here to fix this issue (now it's just the same behavior as what it used to be). If we want to add `trim` behavior before passing `minDate` / `maxDate` to `date-table`, maybe that should be another issue?

Reply:
Actually, I mean resetting the time to 00:00:00 before passing minDate/maxDate to date-table.

date-table checks date equality in a naive way, so it can not handle non 00:00:00 time. But the rest of date-range panel should know about the selected time.

Like this:
```
```

Reply:
That makes sense. Thx.

Reply:
Seems it is a little complex to realize this.

1. I have to reset to `12:00:00` to make it work (I'm still looking into why), which will change the default behavior on pick.

2. There is some "reference equal" issue, which means, when `resetTime` returns a new `Date` object (even with the same time value), it breaks.

I'll spend time on it later. If you have any ideas about the above problems, plz hint me.

Reply:
Interesting. Might have something to do with legacy codes. I suggest you keep current solution, it works. :)

Reply:
Agree that.